resources/scores.txt

```diff
@@ -1 +1 @@
-
+70=Pascal
```

Comment:
Please do not include any pre-defined scores in this file, since a player should start with no scores to compete with.
Delete the content for this file

Reply:
@Isamade Only this small change and we can merge this PR